core/shared/src/main/scala/matryoshka/package.scala

```diff
@@ -94,58 +94,52 @@ package object matryoshka {
   type ElgotCoalgebra[E[_], F[_], A]        = A => E[F[A]] // ElgotCoalgebraM[E, Id, F, A]
 
   /** Transform a structure `F` containing values in `W`, to a structure `G`, 
-    * in bottom-up fashion, accumulating effects in the monad `M`.
-    * @group algebras 
-    */
-  type GAlgebraicTransformM[T[_[_]], W[_], M[_], F[_], G[_]] = F[W[T[G]]] => M[G[T[G]]]
-  /** Transform a structure `F`, to a structure `G`, in bottom-up fashion, 
-    * accumulating effects in the monad `M`.
+    * in bottom-up fashion.
     * @group algebras 
     */
-  type AlgebraicTransformM[T[_[_]], M[_], F[_], G[_]]        = F[T[G]] => M[G[T[G]]] // GAlgebraicTransformM[T, Id, M, F, G]
+  type AlgebraicGTransformM[W[_], M[_], T, F[_], G[_]] = F[W[T]] => M[G[T]]
+
   /** Transform a structure `F` containing values in `W`, to a structure `G`, 
     * in bottom-up fashion.
     * @group algebras 
     */
-  type GAlgebraicTransform[T[_[_]], W[_], F[_], G[_]]        = F[W[T[G]]] => G[T[G]] // GAlgebraicTransformM[T, W, Id, F, G]
-  /** Transform a structure `F` to a structure `G`, in bottom-up fashion.
-    * @group algebras 
-    */
-  type AlgebraicTransform[T[_[_]], F[_], G[_]]               = F[T[G]] => G[T[G]]    // GAlgebraicTransformM[T, Id, Id, F, G]
+  type AlgebraicGTransform[W[_], T, F[_], G[_]]        = F[W[T]] => G[T]
```

Comment:
These new aliases just use the new proper type approach. It means the differ a bit (e.g., the types of `AlgebraicTransform` and `CoalgebraicTransform` are now the same, so they’re just `Transform`).